presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoPreparedStatement.java

```diff
@@ -80,6 +80,14 @@
         super.execute(format("PREPARE %s FROM %s", statementName, sql));
     }
 
+    @Override
+    public void close()
+            throws SQLException
+    {
+        super.execute(format("DEALLOCATE PREPARE %s", statementName));
```

Comment:
What should be the behavior if this throws? Should we still call `super.close()` anyway?

Reply:
indeed, it would be good to call super.close(). What is the cleanest way to do this, without swallowing exceptions? (can't use `Closer` due to `SQLException`)